src/app/pages/BibPage.jsx

```diff
@@ -64,8 +64,20 @@ export const BibPage = (
 
   const allElectronicLocatorsWithAeon = bib.electronicResources
   const { eResources: eResourcesWithoutAeon } = getElectronicResources(bib);
-  const isOnlyElectronicResources = bib.items.length === 0 && bib.electronicResources && bib.electronicResources.length > 0
-  const hasPhysicalItems = !isOnlyElectronicResources && bib.items.length > 0
+
+  const hasElectronicResources = bib.electronicResources && bib.electronicResources.length > 0
+  let numPhysicalItems
+  if (bib.numItemsTotal !== undefined) {
+    numPhysicalItems = bib.numItemsTotal
+  } else if (hasElectronicResources) {
```

Comment:
can we add some comments here describing why not having numitemstotal and having electronic resources necessitates a decrement?